ptero_lsf/implementation/celery_tasks/lsf_task.py

```diff
@@ -63,12 +63,15 @@ def run(self, job_id, lsf_job_id):
             backend = celery.current_app.factory.create_backend()
             backend.kill_lsf_job(lsf_job_id)
         except Exception as exc:
-            delay = DELAYS[self.request.retries]
-            LOG.exception("Exception while trying to kill job (%s) with lsf "
-                    "job id (%s) retrying in %s seconds.  Attempt %d of %d.",
-                    job_id, lsf_job_id, delay, self.request.retries + 1,
-                    self.max_retries + 1, extra={'jobId': job_id})
-            self.retry(exc=exc, countdown=delay)
+            if "Job has already finished" in exc.message:
```

Comment:
We usually get back an [`LSFBindingException`](https://github.com/genome/lsf-python/blob/c1ec25a096309b9b9d7992c19dcacdd2bf9307b5/lsf/exceptions.py#L61) at this point (and not one of the other types in that file).  Maybe it's better to have a separate `except` for those that then does this string matching?  But we'd still want to catch `Exception` generally for the logging bit after that.